packages/jest-expo/package.json

```diff
@@ -27,13 +27,18 @@
     "preset": "jest-expo/universal"
   },
   "peerDependencies": {
-    "react": "*",
+    "react": "~16.8.3",
     "react-native": "*",
     "react-native-web": "*"
   },
   "dependencies": {
     "babel-jest": "^24.7.1",
+    "enzyme": "^3.9.0",
+    "enzyme-adapter-react-16": "^1.12.1",
```

Comment:
my sense is that we don't yet want to commit to supporting enzyme as much as we are supporting jest, and it would be better to split out the enzyme feature into a separate package we can deprecate if needed